external-api/src/v1/controllers/party-db.controller.ts

```diff
@@ -2,10 +2,10 @@
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
Appreciate not touched in this PR, can you remove these inline ESLint please (unless removal creeps the scope).

Reply:
Removed and resolved the resulting lint issues :)